### PR TITLE
SEO-3569 ASP.NET MVC Site Audit Issues

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Have changed title, description, H1 and named images.

Regards,
Yvone.

### CODE REVIEW

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: aspnetmvc/Autocomplete/How-to.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-title: How to
-description: How to
+title: How To Section with ASP.NET MVC TreeView Control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Yvone-Atieno) [reply aspnetmvc/Autocomplete/How-to.md:3]: updated

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

File: aspnetmvc/Button/Overview.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Button| Button | ASP.NET MVC | Syncfusion
-description: overview
+title: Overview in ASP.NET MVC Button Control | Syncfusion
```
Comment (Aishwarya-Ganesan) [inline aspnetmvc/Button/Overview.md:3]: Overview of ASP.NET MVC Button Control - title

File: aspnetmvc/Button/Overview.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+description: Learn here more about Overview in Syncfusion Essential ASP.NET MVC Button Control, its elements, and more.
 platform: ejmvc
 control: Button
 documentation: ug
 ---
 
-# Button
+# Overview in ASP.NET MVC Button
```
Comment (Aishwarya-Ganesan) [inline aspnetmvc/Button/Overview.md:10]: Overview of ASP.NET MVC Button

File: aspnetmvc/Gantt/Columns.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Columns | Gantt | ASP.NET MVC | Syncfusion
```
Comment (Aishwarya-Ganesan) [inline aspnetmvc/Gantt/Columns.md:3]: here title is not changed?

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: aspnetmvc/Button/Overview.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Button| Button | ASP.NET MVC | Syncfusion
-description: overview
+title: Overview in ASP.NET MVC Button Control | Syncfusion
+description: Learn here more about Overview in Syncfusion Essential ASP.NET MVC Button Control, its elements, and more.
```
[the remark this answers is not in the document]
Comment (Yvone-Atieno) [reply aspnetmvc/Button/Overview.md:4]: changed

File: aspnetmvc/Button/Overview.md (hunk 3 above)
Comment (Yvone-Atieno) [reply aspnetmvc/Button/Overview.md:10]: changed

**Pull request merged**